src/presentation/pages/Settings.vue

```diff
@@ -1,14 +1,36 @@
 <template>
   <h1>{{ t('settings.title') }}</h1>
   <ThemeButton />
+  <Button
+    :text="t('auth.logout')"
+    type="primary"
+    :icon="IconUnlink"
+    @click="logout"
+  />
 </template>
 
 <script lang="ts" setup>
 import { useI18n } from 'vue-i18n';
-import ThemeButton from '@/presentation/components/theme/ThemeButton.vue';
+import ThemeButton from '../components/theme/ThemeButton.vue';
+import Button from '../components/button/Button.vue';
+import { IconUnlink } from '@codexteam/icons';
+import { useRouter } from 'vue-router';
+import useAuth from '@/application/services/useAuth';
 
 const { t } = useI18n();
-</script>
+const router = useRouter();
+const { logoutTheUser } = useAuth();
 
+/**
+ * Function to logout the user by deleting the refresh  token from the local storage and redirecting to the main page
+ */
+async function logout() {
+  await logoutTheUser().then(() => {
+    router.push({ path: '/' }).then(() => {
+      window.location.reload();
```

Comment:
why do we need to hard reload a page? It's better to "unauthorize" auth-related staff. At least `notesApiTransport`. Just like we do after authorisation but with opposite actions. Check the `todo` in auth.sevice.ts / logout method

Reply:
Even after I removed the token from the local storage, the change was not apparent until the hard reload. I could access profile settings, profile avatar was still there, etc